### PR TITLE
Core: Update translations

### PR DESCRIPTION
**Pull request by kasperg**

### CODE REVIEW

File: translations/da.po [hunk 1]
```diff
@@ -47949,8 +47949,8 @@ msgid "Create user"
 msgstr "Opret bruger"
 
 #: /gatewayf/callback?destination=ding_frontpage?eduPersonTargetedID=WAYF-DK-e767c06edaad0f8d23e785bbc470e7f37ad03e8d&mail=&logintype=nemlogin
-msgid "You do not have an user account at the public library. If you wish you can create an user account below. You are still logged in at WAYF, so you can !logout to logout of WAYF."
-msgstr "<p>Du er ikke låner ved biblioteket. Hvis du ønsker at oprette dig som låner kan du gøre det nedenfor.</p>\r<p>Du er stadig logget ind ved NemID, så du kan !logout for at logge ud.</p>"
+msgid "You do not have an user account at the public library. If you wish to create an user account !user_url . You are still logged in at WAYF, so you can !logout to logout of WAYF."
+msgstr "<p>Du er ikke låner ved biblioteket. Hvis du ønsker at oprette dig som låner kan du !user_url.</p>\r p>Du er stadig logget ind ved NemID, så du kan !logout for at logge ud.</p>"
```
Comment (guddo) [inline translations/da.po]: There seems to be a typo here.

File: translations/fields_da.po [hunk 2]
```diff
@@ -1464,3 +1464,11 @@ msgstr "Privat - Kun du har adgang til denne liste"
...
 msgid "Rate this"
 msgstr "Bedøm dette materiale"
+
+#: field:field_ding_event_price:ding_event:description
+msgctxt "field_ding_event_price:ding_event:description"
+msgid ""
+"Events show the value ”Free” if text field is set to 0.\n"
+"Tip: Indicate different ticket prices for more than one audience group "
+"by entering a lowest – highest price in text field fx. “30 - 50”"
+msgstr "Arrangementer viser værdien "Gratis" hvis tekstfeltet er sat til 0. Ved flere billettyper kan det angives ved at indsætte laveste - højeste pris i tekstfeltet f. eks. "30 - 50""
```
Comment (guddo) [inline translations/fields_da.po]: `"` and `”` is different characters.

File: translations/fields_da.po [hunk 3]
```diff
@@ -1038,7 +1038,7 @@ msgstr "Place2book Billetter"
 #: field:field_place2book_tickets:ding_event:description
 msgctxt "field_place2book_tickets:ding_event:description"
 msgid "Placeholder for Place2book ticket information and/or order link"
-msgstr ""
+msgstr "Pladsholder for Place2book billetinformation og/eller ordre link. VÆRDIEN "PLACEHOLDER" BØR IKKE ÆNDRES!"
```
Comment (guddo) [inline translations/fields_da.po]: Better use `”` or `'` if you wanna quote something within this `msgstr`.

File: translations/da.po [hunk 4]
```diff
@@ -66629,4 +66629,12 @@ msgid ""
 "loans older than 30 days will be permanently deleted."
 msgstr ""
 "Log ind og rediger din brugerprofil for at give tilladelse til at gemme dine lån ud over 30 dage. Du kan altid fjerne tilladelsen igen hvis du fortryder. "
-"Hvis du vælger at fjerne tilladelsen vil information om lån ældre end 30 dage blive slettet permanent."
\ No newline at end of file
+"Hvis du vælger at fjerne tilladelsen vil information om lån ældre end 30 dage blive slettet permanent."
+
+#: /index.php
+msgid "Events show the value ”Free” if text field is set to 0.\nTip: Indicate different ticket prices for more than one audience group by entering a lowest – highest price in text field fx. “30 - 50”"
+msgstr "Arrangementer viser værdien "Gratis" hvis tekstfeltet er sat til 0. Ved flere billettyper kan det angives ved at indsætte laveste - højeste pris i tekstfeltet f. eks. "30 - 50""
```
Comment (guddo) [inline translations/da.po]: `"` and `”` is different characters.

**Comment by kasperg** [comment]
Thanks for the comments @guddo. They should be fixed now.

With these syntax problems in the `.po` file I wonder why the installer did not complain.

**Comment by guddo** [comment]
Yw. You missed `"Gratis"` here: https://github.com/ding2/ding2/pull/518/files#diff-e45508e5f2bf7e01992b1120a4d3e34fR66636

**Comment by kasperg** [comment]
😭 . See #519.

**Pull request merged**